openwisp_monitoring/device/models.py

```diff
@@ -1,22 +1,21 @@
 from django.contrib.contenttypes.fields import GenericRelation
-from swapper import get_model_name, swappable_setting
+from swapper import get_model_name, load_model, swappable_setting
 
-from ..utils import load_model_patched
 from .base.models import AbstractDeviceData, AbstractDeviceMonitoring
 
-BaseDevice = load_model_patched('config', 'Device', require_ready=False)
+BaseDevice = load_model("config", "Device", require_ready=False)
```

Comment:
why are you changing the quoting style? That's an unwelcome change.
If it's because of the black formatter, please use `openwisp-qa-format` to reformat code as indicated in our guidelines.

Reply:
Sorry, my bad. I've already used `openwisp-qa-format` for formatiing. Maybe it is due to the version of black I'm using.